### PR TITLE
feat: add create list items endpoint

### PR DESCRIPTION
**Pull request by johnytiago**
There's a lot of endpoints that are not currently covered by the SDK. 
Would you be willing to accept this PR to extend that coverage?

I've searched other language SDKs and I couldn't find an implementation of this endpoint so I could copy the interface.
Not sure if this would be the naming you're after. If not, let me know and I can update it.

### CODE REVIEW

File: src/command/services/command-create-list-item.service.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+export const CommandCreateListItemService = {
+  call: (
+    controller,
+    options: ListItemPayload,
+    configuration: Configuration
+  ) => {
+    ValidatorPresentService.call(options, [
+      'list_id',
+      'primary_value',
```
Comment (bartes) [inline src/command/services/command-create-list-item.service.ts:14]: could we validate the `list_id` path param?  We want to keep other validation on the API until we prepare some more robust validation functionality.

File: test/api/services/api-create-list-item.service.test.ts [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  const sampleRequestData = {
+    list_id: 'e6baae3a-0636-441a-ba4f-c73f266c7a17',
+    primary_value: 'A04t7AcfSA69cBTTusx0RQ',
+    secondary_value: '2ee938c8-24c2-4c26-9d25-19511dd75029',
+    comment: 'Fradulent user found through automated inspection',
+    author: {
+      type: '$other' as ListItemAuthorType,
+      identifier: 'Security bot',
+    },
+    mode: 'error' as ListItemMode,
```
Comment (bartes) [inline test/api/services/api-create-list-item.service.test.ts:16]: could we change it t `$error` ?

File: test/command/services/command-create-list-item.service.test.ts [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    const options = {
+      list_id: 'e6baae3a-0636-441a-ba4f-c73f266c7a17',
+      primary_value: 'A04t7AcfSA69cBTTusx0RQ',
+      secondary_value: '2ee938c8-24c2-4c26-9d25-19511dd75029',
+      comment: 'Fradulent user found through automated inspection',
+      author: {
+        type: '$other' as ListItemAuthorType,
+        identifier: 'Security bot',
+      },
+      mode: 'error' as ListItemMode,
```
Comment (bartes) [inline test/command/services/command-create-list-item.service.test.ts:28]: could we add '$' prefix here ?

**Review by bartes — COMMENTED** [review]
hi @johnytiago ,

Thank you for the great PR.  🤝  .

Yes, the functionality of the lists is not yet in our SDKs, but it's in our roadmap. 
We plan to prepare some more robust solutions for the SDKs

I added some tiny comments, but besides that, all looks good. 🚀

File: src/command/services/command-create-list-item.service.ts (hunk 1 above)
Comment (johnytiago) [reply src/command/services/command-create-list-item.service.ts:14]: Right now I'm only asserting it is defined. What other validations you suggest I do?

**Review by bartes — APPROVED** [review]
🚀 👍

**Pull request merged**